spec.html

```diff
@@ -20739,6 +21579,26 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. Set _value_.[[SourceText]] to the source text matched by |ClassExpression|.
         1. Return _value_.
       </emu-alg>
+      <emu-grammar>ClassElementName : PropertyName</emu-grammar>
+      <emu-alg>
+        1. Return the result of evaluating PropertyName.
+      </emu-alg>
+      <emu-grammar>ClassElementName : PrivateIdentifier</emu-grammar>
+      <emu-alg>
+        1. Let _privateIdentifier_ be StringValue of |PrivateIdentifier|.
+        1. Let _scope_ be the running execution context's PrivateEnvironment.
+        1. Let _scopeEnvRec_ be _scope_'s EnvironmentRecord.
```

Comment:
```suggestion
        1. Let _scopeEnvRec_ be the running execution context's PrivateEnvironment.
```
(but a better name would probably be `_privateEnvRec_`)